Latest version not installing monotonic

Our CI runs are failing due to missing `monotonic` module after using `pip install spectacles` to set up our environment. Seeing this locally as well.

Failure:
```
Run bash ../etc/ci.run-spectacles-lookml.sh prod
Running Spectacles: LookML validation
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.9.18/x64/bin/spectacles", line 5, in <module>
    from spectacles.cli import main
  File "/opt/hostedtoolcache/Python/3.9.[18](https://github.com/replicatedhq/data/actions/runs/7738160085/job/21098442542#step:5:19)/x64/lib/python3.9/site-packages/spectacles/cli.py", line [19](https://github.com/replicatedhq/data/actions/runs/7738160085/job/21098442542#step:5:20), in <module>
    import spectacles.tracking as tracking
  File "/opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/site-packages/spectacles/tracking.py", line 5, in <module>
    import analytics
  File "/opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/site-packages/analytics/__init__.py", line 3, in <module>
    from analytics.client import Client
  File "/opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/site-packages/analytics/client.py", line 11, in <module>
    from analytics.consumer import Consumer
  File "/opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/site-packages/analytics/consumer.py", line 3, in <module>
    import monotonic
ModuleNotFoundError: No module named 'monotonic'
Error: Process completed with exit code 1.
```

Will paste `pip install spectacles` output into a comment

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "spectacles"
-version = "2.4.0"
+version = "2.4.1"
 description = "A command-line, continuous integration tool for Looker and LookML."
 authors = ["Spectacles <hello@spectacles.dev>"]
 license = "MIT"

--- a/spectacles/tracking.py
+++ b/spectacles/tracking.py
@@ -2,7 +2,7 @@ import hashlib
 import uuid
 from typing import Optional
 
-import analytics
+import segment.analytics as analytics
 
 analytics.write_key = "QnqzXWlqkmgDSm7X2qFDrxx3LGCW7Rba"
 

--- a/tests/unit/test_tracking.py
+++ b/tests/unit/test_tracking.py
@@ -13,7 +13,7 @@ def test_anonymise_project() -> None:
     assert hashed_url == "6e72a69d5c5cca8f0400338441c022e4"
 
 
-@patch("analytics.track")
+@patch("segment.analytics.track")
 def test_track_invocation_start_sql(mock_track: MagicMock) -> None:
     invocation_id = tracking.track_invocation_start(
         "https://organisation.looker.com", "sql", "123456", "test_project"
@@ -31,7 +31,7 @@ def test_track_invocation_start_sql(mock_track: MagicMock) -> None:
     assert invocation_id == "123456"
 
 
-@patch("analytics.track")
+@patch("segment.analytics.track")
 def test_track_invocation_start_assert(mock_track: MagicMock) -> None:
     invocation_id = tracking.track_invocation_start(
         "https://organisation.looker.com", "assert", "123456", "test_project"
@@ -49,7 +49,7 @@ def test_track_invocation_start_assert(mock_track: MagicMock) -> None:
     assert invocation_id == "123456"
 
 
-@patch("analytics.track")
+@patch("segment.analytics.track")
 def test_track_invocation_start_connect(mock_track: MagicMock) -> None:
     invocation_id = tracking.track_invocation_start(
         "https://organisation.looker.com", "assert", "123456"
@@ -67,7 +67,7 @@ def test_track_invocation_start_connect(mock_track: MagicMock) -> None:
     assert invocation_id == "123456"
 
 
-@patch("analytics.track")
+@patch("segment.analytics.track")
 def test_track_invocation_end_sql(mock_track: MagicMock) -> None:
     tracking.track_invocation_end(
         "https://organisation.looker.com", "sql", "123456", "test_project"
@@ -84,7 +84,7 @@ def test_track_invocation_end_sql(mock_track: MagicMock) -> None:
     )
 
 
-@patch("analytics.track")
+@patch("segment.analytics.track")
 def test_track_invocation_end_assert(mock_track: MagicMock) -> None:
     tracking.track_invocation_end(
         "https://organisation.looker.com", "assert", "123456", "test_project"
@@ -101,7 +101,7 @@ def test_track_invocation_end_assert(mock_track: MagicMock) -> None:
     )
 
 
-@patch("analytics.track")
+@patch("segment.analytics.track")
 def test_track_invocation_end_connect(mock_track: MagicMock) -> None:
     tracking.track_invocation_end("https://organisation.looker.com", "assert", "123456")
     mock_track.assert_called_once_with(

